react/components/SelectionListOrderBy.js

```diff
@@ -37,39 +51,34 @@ class SelectionListOrderBy extends Component {
     })
   }
 
+  getOrderByTitle = order => {
+    const { options, intl } = this.props
+    let title = intl.formatMessage({ id: 'search-result.orderby.title' })
+    if (location.search.includes('order')) {
+      options.forEach(opt => {
+        if (opt.value === order) {
+          title = opt.label
+        }
+      })
+    }
+    return title
+  }
+
   render() {
-    const { intl, options, orderBy } = this.props
+    const { orderBy } = this.props
+    const { showDropdown } = this.state
+    const btClass = `bg-base c-on-base t-action--small ml-auto ${showDropdown ? 'vtex-dropdown-btn__active b--muted-4' : 'vtex-dropdown-btn'}`
 
     return (
-      <Popup
-        title={intl.formatMessage({ id: 'search-result.orderby.title' })}
-        id="orderby"
-      >
-        <div className="vtex-orderby-popup">
-          {options.map(opt => {
-            const active = orderBy === opt.value
-
-            return (
-              <div
-                key={opt.label}
-                className={classNames('vtex-orderby__item pointer pv3 ph7 bb b--muted-4', {
-                  'vtex-orderby__item--active t-body c-on-base': active,
-                  'c-disabled': !active,
-                })}
-                onClick={this.handleSelect(opt)}
-              >
-                {opt.label}
-
-                {active && (
-                  <span className="vtex-orderby__item-icon fr">
-                    <CheckTick />
-                  </span>
-                )}
-              </div>
-            )
-          })}
+      <div align="center|justify" className="vtex-dropdown flex justify-center flex-auto pt1">
+        <button onClick={this.handleDropdownBtClick} className={btClass}>
+          {this.getOrderByTitle(orderBy)}
+          <span className="vtex-orderby__item-icon fr v-mid"><Arrow size={16} /></span>
+        </button>
+        <div className={`vtex-dropdown-content bg-base ${showDropdown ? 'b--muted-4 vtex-dropdown-content__active' : ''}`}>
```

Comment:
rename the `vtex-dropdown-content` to `vtex-dropdown__content`

and rename the `vtex-dropdown-content__active` to `vtex-dropdown__content--active`